server/controllers.py

```diff
@@ -60,4 +59,18 @@ def get_emotion_by_id(emotion_id : int):
     return {"emotion": emotions[emotion_id]} 
 
 if __name__ == "__main__":
+    
+    model = CNN()
+    optimizer = optim.Adam(model.parameters())
+
+    # Load a model
+    checkpoint = torch.load('/home/hovhannes/Desktop/FacialExpressionRecognition/model/checkpoints/model.pkl')
+    model.load_state_dict(checkpoint['model_state_dict'])
+    optimizer.load_state_dict(checkpoint['optimizer_state_dict'])
+    epoch = checkpoint['epochs']
+
+    # Save this in train stage
+    # accuracy_on_validation_set = evaluate(model, val_loader)
+    # accuracy_on_train_set = evaluate(model, train_loader)
```

Comment:
Please confirm: to make a prediction you always need a DataLoader?

Reply:
For making a prediction, we need to image pass through our transform. After that we can give it to our model, without any DataLoader.